mongoInit.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+db.runCommand({
```

Comment:
I'd suggest that a user entity and repository be created and then this call those to create the necessary records(reduce duplication, type safety etc. etc.).

Reply:
Just had a thought here. I think you can get access to the nest application context outside of nest [like so](https://docs.nestjs.com/application-context) and from there you might be able to grab an instance of a service and create your user that way?

Reply:
Since this is very specifically [a script to be executed in the mongo shell](https://docs.mongodb.com/manual/tutorial/write-scripts-for-the-mongo-shell/) it really can't be decoupled from the mongo implementation (AFAIK you can't even `import` or `require` external libraries within the mongo shell).

As it happens, I converted it to a shell script in 4cf25b4 -- which allows us to actually use the container's environment variables. That should make it much easier to get set up.

Reply:
I was more thinking of calling the actual app via CLI and therefore re-using any database stuff contained within it rather than having additional scripts to call and maintain. Not sure what you think of that as an idea....

Reply:
Just to recap on this for future reference - we decided we can't do this because the external script is required to provision the access and DB for the app so that the app can connect in the first place....